### PR TITLE
daily-status.py: Include number of absent workflows & clients in subject

### LINKED ISSUE #160: Add "ABSENT" count of clients for which no test runs were detected
apparently one of the clients was "read-only" in few past days and because email just had "PASSED" (although with reduced count).  To alert to the fact that some client(s) might have had missed providing any results, let's
- do list a client even if no results -- then just say "ABSENT"
- include ABSENT count into subject line

### PR DESCRIPTION
**Pull request by jwodder**
Closes #160.

### CODE REVIEW

File: .github/workflows/tools/daily-status.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
 #!/usr/bin/env python3
 from __future__ import annotations
 
-__python_requires__ = "~= 3.8"
+__python_requires__ = ">= 3.8"
```
Comment (yarikoptic) [inline .github/workflows/tools/daily-status.py:4]: just curious -- why this change?

Comment (jwodder) [reply .github/workflows/tools/daily-status.py:4]: Specifying an upper bound on `python_requires`/`Requires-Python` is considered bad practice.  It doesn't make a difference here, as this is just for human-readable documentation, but the documentation should still reflect best practices.

Comment (yarikoptic) [reply .github/workflows/tools/daily-status.py:4]: do you remember where such "bad practice" described? 
From my experience if anything should be upper bound is Python version itself with it changing behavior of such rudimentary operations such as `/` while hopping from 2 to 3, and thus leading to not just crashes but to silently producing incorrect results.

Comment (jwodder) [reply .github/workflows/tools/daily-status.py:4]: The Python devs have stated (somewhere) that the Python 3 → Python 4 transition will be more like the 1 → 2 transition than like 2 → 3, i.e., with much more backwards compatibility.  See https://discuss.python.org/t/requires-python-upper-limits/12663 for too much analysis of the problems with upper Python bounds.
